x-pack/plugins/watcher/public/sections/watch_edit/components/watch_edit.tsx

```diff
@@ -155,6 +156,8 @@ export const WatchEdit = ({
 
   if (watch.type === WATCH_TYPES.THRESHOLD) {
     EditComponent = ThresholdWatchEdit;
+  } else if (watch.type === WATCH_TYPES.MONITORING) {
```

Comment:
Nit: this could be a switch statement since all conditions are comparing against `watch.type`.